### DISCUSSION TITLE
Add an "Unpublish" button to version-enabled collections

### DISCUSSION THREAD

**Discussion by yandodov** [payloadcms/payload#2981] [Feature Requests & Ideas]
Currently, once you initially publish a version-enabled document, you can only publish further edits. You can't completely unpublish a document and make it unavailable again in an easy way.

The only way I've found is to go to the document's versions, select the draft version, and click "Restore this version". That seems way too complicated, though.

If a user mistakenly publishes a blog post, for example, and wants to make it unavailable again, they should have an easy way to do so — an "Unpublish" button.

**Comment by jmikrut** [comment]
Hey @hdodov — we actually _**do**_ have an `unpublish` button:

![Screen Shot 2023-07-07 at 8 34 18 AM](https://github.com/payloadcms/payload/assets/6651896/b88f415f-1fa1-4d82-9692-0005b4dec293)

Is this missing for you?

**Reply by yandodov** [reply]
I didn't notice that button, you're right. But it only appears when there is no pending draft. If you have an active draft, you only get a "Revert to published" button:

![image](https://github.com/payloadcms/payload/assets/5570098/f7701bb5-e858-4962-8722-363b597c6095)

So to completely unpublish a document, you need to revert it to the published version first, which is odd and is bad UX in my opinion. If it wasn't confusing, I wouldn't have opened an issue.

When there's an active draft, I think that both "Revert to published" and "Unpublish" should be available.

**Reply by jmikrut** [reply]
Ah I see! And I agree. We will make this happen and thanks for the request. Makes a lot of sense!
